### ISSUE TITLE
possibility to set a tag ?

### ISSUE THREAD

**Issue by aacebedo** [flameshot-org/flameshot#3443]
### Feature Description

Last tag has been put more than one year ago. Support of wayland compositors has been greatly improved since.
As I am using nixos I would like to update the central repo but I need an official tag for that.
Is it possible to add a tag?

**Comment by jflambert** [comment]
I'm curious what's holding this back?

**Comment by mmahmoudian** [comment]
> I'm curious what's holding this back? 

Simple answer: time. We have discussed on Flameshot matrix channel about preparation for a release. The bottleneck is as always "time". We have to solve the weblate conflict, merge some of the ready PRs, and then have a feature freeze, then a Release Candidate, then release.

But all help are highly appreciated, especially reviewing the open PRs, testing them. The more eyes are on the code and documentation, the better the project gets.

**Comment by jflambert** [comment]
Fair enough! Been a daily user on Ubuntu 22.04 for a few years, can't complain, was just surprised at the lack of updates. Good luck with contributions.
